src/mesh.c

```
@@ -6,381 +6,360 @@
+#define ROUTE_INVALID   0
+#define ROUTE_DIRECT    1
+#define ROUTE_INDIRECT  2
 
 /* Radio callbacks */
-static void mesh_rx_handler(uint8_t *data, size_t len, int16_t rssi, int8_t snr);
+static void mesh_rx_handler(uint8_t* data, size_t len, int16_t rssi, int8_t snr);
 static void mesh_tx_handler(void);
 static void mesh_error_handler(uint16_t error);
 
+/* Group management helpers called from examples/app-level code */
+int mesh_send_group_join_invite(uint16_t leader_id, uint16_t member_id, uint16_t group_id);
```

Clang (14.0.6)

[warning] redundant 'mesh_send_group_join_invite' declaration

(readability-redundant-declaration)